### PR TITLE
Improve readme of `depth_guided_stable_diffusion` example

### PR DESCRIPTION
**Pull request by BirgerMoell**
### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [app.rerun.io](https://app.rerun.io/pr/5502/index.html)
  * Using examples from latest `main` build: [app.rerun.io](https://app.rerun.io/pr/5502/index.html?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [app.rerun.io](https://app.rerun.io/pr/5502/index.html?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/5502)
- [Docs preview](https://rerun.io/preview/3f3a6ad3fb1dc7eefb079693f211443467cd3f04/docs) 
- [Examples preview](https://rerun.io/preview/3f3a6ad3fb1dc7eefb079693f211443467cd3f04/examples) 
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

**Comment by teh-cmc** [comment]
(Drafting since the title says it's a draft)

File: examples/python/depth_guided_stable_diffusion/README.md [hunk 1]
```diff
@@ -1,26 +1,111 @@
...
+Leverage [Depth Guided Stable Diffusion](https://github.com/Stability-AI/stablediffusion?tab=readme-ov-file#depth-conditional-stable-diffusion) to generate images with enhanced depth perception. This method integrates depth maps to guide the Stable Diffusion model, creating more visually compelling and contextually accurate images.
+
+## Used Rerun Types
+[`Image`](https://www.rerun.io/docs/reference/types/archetypes/image), [`Tensor`](https://www.rerun.io/docs/reference/types/archetypes/tensor), [`DepthImage`](https://www.rerun.io/docs/reference/types/archetypes/depth_image)
+
+## Background
+Depth Guided Stable Diffusion enriches the image generation process by incorporating depth information, providing a unique way to control the spatial composition of generated images. This approach allows for more nuanced and layered creations, making it especially useful for scenes requiring a sense of three-dimensionality.
+
+# Logging and Visualizing with Rerun
+The visualizations in this example were created with the Rerun SDK, demonstrating the integration of depth information in the Stable Diffusion image generation process. Here is the code for generating the visualisation in Rerun.
```
Comment (emilk) [inline examples/python/depth_guided_stable_diffusion/README.md:27]: Note: the reason the spellcheck complains is that we prefer american english, i.e. "visualization". You can run `typos -w` to fix this (after installing `typos`: https://github.com/crate-ci/typos)

File: examples/python/depth_guided_stable_diffusion/README.md [hunk 2]
```diff
@@ -1,26 +1,111 @@
...
+
+## Prompt
+Visualising the prompt and negative prompt
+```python
+rr.log("prompt/text", rr.TextLog(prompt))
+rr.log("prompt/text_negative", rr.TextLog(negative_prompt))
```
Comment (nikolausWest) [inline examples/python/depth_guided_stable_diffusion/README.md:33]:
```suggestion
rr.log("prompt/text", rr.TextDocument(prompt))
rr.log("prompt/text_negative", rr.TextDocument(negative_prompt))
```

**Review by nikolausWest — APPROVED** [review]
Looks good to me! All that's left is getting it all the way to green CI

File: examples/python/depth_guided_stable_diffusion/README.md [hunk 3]
```diff
@@ -1,26 +1,111 @@
...
   <source media="(max-width: 1200px)" srcset="https://static.rerun.io/depth_guided_stable_diffusion/a85516aba09f72649517891d767e15383ce7f4ea/1200w.png">
-  <img src="https://static.rerun.io/depth_guided_stable_diffusion/a85516aba09f72649517891d767e15383ce7f4ea/full.png" alt="Depth-guided stable diffusion screenshot">
+  <img src="https://static.rerun.io/depth_guided_stable_diffusion/a85516aba09f72649517891d767e15383ce7f4ea/full.png" alt="Depth-guided stable diffusion example">
 </picture>
 
-A more elaborate example running Depth Guided Stable Diffusion 2.0.
+Leverage [Depth Guided Stable Diffusion](https://github.com/Stability-AI/stablediffusion?tab=readme-ov-file#depth-conditional-stable-diffusion) to generate images with enhanced depth perception. This method integrates depth maps to guide the Stable Diffusion model, creating more visually compelling and contextually accurate images.
+
+## Used Rerun Types
+[`Image`](https://www.rerun.io/docs/reference/types/archetypes/image), [`Tensor`](https://www.rerun.io/docs/reference/types/archetypes/tensor), [`DepthImage`](https://www.rerun.io/docs/reference/types/archetypes/depth_image)
```
Comment (nikolausWest) [inline examples/python/depth_guided_stable_diffusion/README.md:21]: You missed `rr.TextLog` although, as  I suggested below I think you should also update that code to use `rr.TextDocument`

Comment (emilk) [reply examples/python/depth_guided_stable_diffusion/README.md:21]: It would be nice to auto-generate this list 🤔

**Comment by BirgerMoell** [comment]
Made a clean pull request here.
https://github.com/rerun-io/rerun/pull/5593

**Comment by emilk** [comment]
Replaced by https://github.com/rerun-io/rerun/pull/5593

**Pull request closed without merging**